src/helpers/checkPluginOpts.ts

```diff
@@ -25,6 +25,10 @@ const checkPluginOpts = (pluginOptions: IPluginOptions): boolean => {
     throw new Error(`\`pluginOptions.serializeFeed\` should be a function of the correct structure.`);
   }
 
+  if (typeof pluginOptions.feedName !== "undefined" && typeof pluginOptions.feedName !== "string") {
+    throw new Error(`\`pluginOptions.feedName\` should be a string used to name file and create file path.`);
```

Comment:
Please could it say `...should be a string used to name feed files and create file path.`

Reply:
Done in same commit where I renamed `feedName` to `feedFilename`